File: Ejercicios/C#/Clases/Persona.cs
public class Persona
{
    public string Nombre;
    public int Edad;

    public Persona()
    {
        Nombre = "Sin nombre";
        Edad = 0;
    }

    public Persona(string nombre, int edad)
    {
        Nombre = nombre;
        Edad = edad;
    }

    public Persona(string nombre)
    {
        Nombre = nombre;
        Edad = 0;
    }

    public Persona(Persona otra)
    {
        Nombre = otra.Nombre;
        Edad = otra.Edad;
    }
}

File: Ejercicios/Java/Clases/Persona.java
public class Persona {
    private String nombre;
    private int edad;

    public Persona() {
        this.nombre = "Sin nombre";
        this.edad = 0;
    }

    public Persona(String nombre, int edad) {
        this.nombre = nombre;
        this.edad = edad;
    }

    public Persona(String nombre) {
        this(nombre, 0);
    }

    public Persona(Persona otra) {
        this.nombre = otra.nombre;
        this.edad = otra.edad;
    }
}
